format/FormatMultiImage.py

```diff
@@ -201,6 +201,7 @@ def get_imageset(
         assert not (as_sequence and lazy), "No lazy support for sequences"
 
         if single_file_indices is not None:
+            assert len(single_file_indices) > 0
```

Comment:
```suggestion
            assert len(single_file_indices)
```
would be functionally equivalent and save 3 characters :)